tests/testProjectClassToR1.m

```diff
@@ -0,0 +1,286 @@
+classdef testProjectClassToR1 < matlab.unittest.TestCase
+
+    properties
+        inputStandardProjectClass
+        inputStandardStruct
+        outputStandardStruct
+        defaultProject
+        outputCustomStruct
+        inputCustomProjectClass
+        modelFile
+        fullPath
+    end
+
+    methods(TestClassSetup)
+        function defineVariables(testCase)
+            testCase.inputStandardProjectClass = "originalDSPCBilayerProjectClass.mat";
+            testCase.inputStandardStruct = "originalDSPCBilayerStructInput.mat";
+            testCase.outputStandardStruct = "originalDSPCBilayerStructOutput.mat";
+            testCase.defaultProject = "defaultProject.mat";
+            testCase.outputCustomStruct = "monolayerVolumeModel.mat";
+            testCase.inputCustomProjectClass = "monolayerVolumeModelProjectClass.mat";
+            testCase.modelFile = "monolayerVolumeCustomLayerModel.m";
+        end
+    end
+
+    methods(TestMethodSetup)
+        function setWorkingFolder(testCase)
+            root = getappdata(0,'root');
+            testCase.fullPath = [root filesep 'tests' filesep 'testProjectClassToR1Conversion'];
+            import matlab.unittest.fixtures.CurrentFolderFixture
+            testCase.applyFixture(CurrentFolderFixture(testCase.fullPath))
+        end
+    end 
+
+     methods(TestMethodTeardown)
+        function testCleanUp(testCase)
+            cd (testCase.fullPath)
+            if exist('newDirectory','dir')
+                rmdir('newDirectory', 's')
+            end
+        end
+    end 
+
+    methods (Test)
+
+        function testFilesExists(testCase)
+            testCase.verifyEqual(exist(testCase.outputStandardStruct,'file'), 2);
+            testCase.verifyEqual(exist(testCase.inputStandardProjectClass,'file'), 2);
+            testCase.verifyEqual(exist(testCase.inputStandardStruct,'file'), 2);
+            testCase.verifyEqual(exist(testCase.defaultProject,'file'), 2);
+            testCase.verifyEqual(exist(testCase.outputCustomStruct,'file'), 2);
+            testCase.verifyEqual(exist(testCase.inputCustomProjectClass,'file'), 2);
+            testCase.verifyEqual(exist(testCase.modelFile,'file'), 2);
+        end
+
+        function testCompareProjectClassToR1(testCase)
+            expected = load(testCase.outputStandardStruct).problem;
+
+            pClass = load(testCase.inputStandardProjectClass).thisProjectClass; 
+            result = projectClassToR1(pClass, "saveProject", false);
+
+            testCase.verifyEqual(result.name, expected.name);
```

Comment:
Why not `testCase.verifyEqual(result, expected);` it should work instead of comparing all the fields?